setup.py

```diff
@@ -23,21 +23,25 @@
     packages=find_packages(exclude=["tests", "tests.*"]),
     python_requires=">=3.6",
     install_requires=required,
```

Comment:
is this a typo?

Reply:
this looks the same as the develop branch

Reply:
I think the typo referred to here is in a following line.  There is another comment pointing it out.